src/modlunky2/sprites/tilecode_extras.py

```diff
@@ -127,9 +127,18 @@ class StickyTrapSheet(BaseSpriteLoader):
     }
 
 
+class PoisonSheet(BaseSpriteLoader):
+    _sprite_sheet_path = Path("static/images/arrows_poison.png")
```

Comment:
This needs to be updated to match the filename (`venom.png`)

Reply:
done